wordpress-mobile/WordPress-iOS
# [Stats Refresh] DWMY Details Async loading and Ghost views

Refs. #12147 

This PR continues the Load Blocks Independently implemented in the DWMY overview, adding the loading block in the Period details.
The loading blocks layout changes based on the cached data. In case there's partial data, only one Ghost row will be added to the bottom of the list.

![stats-dwmy-loading-blocks](https://user-images.githubusercontent.com/912252/67982264-40eaf500-fc1a-11e9-81a8-65c431c2e1db.jpg)

## To test:
• Open Stats -> DWMY from a Site you never opened before (or from a fresh install)
• Play with all the cards that have _View More_ as button and tap it
• You should open the detail view and see the last Ghost row animated
• Disable the feature flag and make sure everything works as before

Update release notes:
